bundle/edu.gemini.horizons.api/src/test/scala/edu/gemini/horizons/server/backend/HS2Spec.scala

```diff
@@ -199,7 +199,7 @@ object HS2Spec extends Specification with ScalaCheck {
         "(1683005400000,2023-May-02 05:30:00 UTC 08:12:07.022 +03:08:06.46 0.577456 1.085775 -1.0 25.601)",
         "(1684569600000,2023-May-20 08:00:00 UTC 08:12:40.294 +03:14:39.1 1.767352 0.701981 -1.0 25.619)",
         "(1686133800000,2023-Jun-07 10:30:00 UTC 08:13:44.373 +03:18:12.92 2.707549 0.271758 -1.0 25.634)",
-        "(1687698000000,2023-Jun-25 13:00:00 UTC 08:15:12.407 +03:18:36.17 3.351083 -0.17043 20.719 25.646)",
```

Comment:
Unrelated.  These ephemeris seems to be drifting.  Maybe we should disable this test.